.github/workflows/ci.yml

```
@@ -46,7 +46,11 @@ jobs:
         with:
           python-version: "3.12"
       - run: pip install -e ".[dev]"
+<<<<<<< HEAD
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)